backend/routes/feedback.ts

```diff
@@ -13,7 +13,7 @@ feedbackRouter.post('/report', async (req: Request, res: Response) => {
       return res.status(200).json({ feedbackData });
     }
     return res.status(500).json({ message: 'Failed to create a report ticket' });
-  } catch (error) {
+  } catch (_error) {
```

Comment:
these and others were not an issue until I upgraded the lint version. Looks like we have a bunch of unused local variables in the code. I added a way to ignore the lint without explicitly turning them off with a comment but by adding a `_`